docs/writing_review_screen.md

```diff
@@ -0,0 +1,240 @@
+
+```yaml
+include:
+  - review.yml
+```
+
+### Step 2: Understanding the Generated Review Screen
+
+A first quick note is that the generated YAML might look different than how you're used to writing YAML files. The generated YAML uses lots of `|-`,
+which keeps the full text after, but removes the last newline. In short, something like this:
+
+```yml
+---
+
+event: |-
+  review_intro
+
+code: |-
+  review_form
+
+```
+
+is the same as this:
+
+```yml
+event: review_intro
+code: |
+  review_form
+```
+
+and I'd recommend changing the YAML to how you normally write it, to be more concise [^1] The remainder of the examples will use the more concise YAML notation.
+
+There are a few sections of the generated review screen YAML:
+
+1. There are the events. These blocks look like this:
+    ```yml
+    event: review_intro
+    code: |
+      review_form
+    ---
+    event: review_prior_case
+    code: |-
+      review_form
+    ---
+    ...
+    ```
+
+    These blocks take each section from your interview and make it so that if a user clicks on them, they are brought to the review page. In our experience, this
+    works best, so that all of the information that is asked for in the interview can be edited in the same place, and the user doesn't have to remember
+    which section they entered it in.
+
+2. The main review block. The block looks like this:
+
+    ```yml
+    id: review screen
+    event: review_form
+    question: |
+      Review your answers
+    review:
+      - Edit: users.revisit
+        button: |
+          **Users**
+
+          % for item in users:
+          - ${ item }
+          % endfor
+      ...
+      - Edit: has_existing_case
+        button: |
+          **Do you have an existing Michigan family court case?**
+
+          Do you have an existing Michigan family court case?: ${ word(yesno(has_existing_case)) }
+    ```
+
+    Most notably is that there is a `review` attribute instead of a `fields`. Each element under the `review` attribute
+    is a different section, generated from screens in the interview, with the subquestion and values of the fields shown.
+
+    Each element in the review can have the following dictionary attributes:
+
+    * `Edit` adds an "Edit" button next to the information on the review screen. It's value is the name of the variable that will be edited when the button is clicked.
+    * `label` lets you change the text on the "Edit" button
+    * `fields` lets you add multiple fields that are re-asked to the user for a section on the review screen.
```

Comment:
I had this in because I thought that `Edit` only took single variables, not the lists that allowed for `recompute` in them, which is something that the generated review screen doesn't use, and is very necessary for correctness in review screens. Pleasantly surprised that `Edit` takes lists of things, so I'll make this simpler